examples/echo-query/main.go

```diff
@@ -160,30 +164,24 @@ func main() {
 		logger.Errorf("No device found. tenant=%s", tenant.Name())
 		os.Exit(-1)
 	}
-	// Select first device
-	device := devices[0]
-	logger.Infof("Selected first device name=%s tenant=%s id=%s", device.Name(), device.Tenant().Name(), device.ID())
-
-	// Setup input
-	var reader io.Reader
-	if *file != "" {
-		f, err := os.Open(*file)
-		if err != nil {
-			panic(err)
+
+	var filteredDevices []sdk.Device
+	//Filter the devices based on the configured regions
+	for _, device := range devices {
```

Comment:
Let's just find the first device that matches the region and use it

Reply:
> Let's just find the first device that matches the region and use it

Fixed